git-branchless/tests/command/test_move.rs

```diff
@@ -361,51 +454,1397 @@ fn test_move_insert_tree() -> eyre::Result<()> {
         git.run(&[
             "move",
             "--on-disk",
-            "--insert",
-            "-s",
-            &test4_oid.to_string(),
+            "--exact",
+            &format!("{} + {} + {}", test3_oid, test5_oid, test7_oid),
             "-d",
             &test1_oid.to_string(),
         ])?;
-        {
-            let (stdout, _stderr) = git.run(&["smartlog"])?;
-            insta::assert_snapshot!(stdout, @r###"
-            :
-            O 62fc20d (master) create test1.txt
-            |
-            @ bf0d52a create test4.txt
-            |\
-            | o 44352d0 create test2.txt
-            |
-            o 0a4a701 create test3.txt
-            "###);
-        }
+        // insta::assert_snapshot!(stderr, @r"");
+
```

Comment:
```suggestion

```

Reply:
Yeah, sorry, there were a lot of WIP artifacts still lingering in here. I've removed a bunch of them, but still need to give this another proof read to make sure I've gotten all of them.